spec/Section 6 -- Execution.md

```diff
@@ -143,6 +146,120 @@ ExecuteMutation(mutation, schema, variableValues, initialValue):
     selection set.
   * Return an unordered map containing {data} and {errors}.
 
+### Subscription
+
+We define an event stream as a sequence of discrete events over time that can be
+observed. An observer of an event stream may cancel observation to avoid stop
+future events.
+
+If the operation is a subscription, the result is an event stream called the
+"Publish Stream" where each event in the stream is called a "Publish Payload".
```

Comment:
I fear "Publish Stream" may be too ambiguous - it could be too easy to confuse it with the underlying stream. How about: "Source Stream" and "Response Stream"?